### PR TITLE
Add Draw command to RenderCommand

### PR DESCRIPTION
**Pull request by lachlansneff**
This adds a `Draw` to the `RenderCommands` enum. Pretty much the same as `DrawIndexed`, just a little different.

Apologies for all the PRs today. This is the end of my backlog 😁

### CODE REVIEW

**Comment by cart** [comment]
Too many PRs is such a good problem to have :heart:

File: crates/bevy_render/src/render_graph/nodes/pass_node.rs [hunk 1]
```diff
@@ -306,6 +313,11 @@ impl DrawState {
...
     }
 
+    pub fn can_draw(&self) -> bool {
+        self.bind_groups.iter().all(|b| b.is_some())
+            && self.vertex_buffers.iter().all(|v| v.is_some())
+    }
+
     pub fn can_draw_indexed(&self) -> bool {
         self.bind_groups.iter().all(|b| b.is_some())
             && self.vertex_buffers.iter().all(|v| v.is_some())
```
Comment (multun) [inline crates/bevy_render/src/render_graph/nodes/pass_node.rs:323]: you can rewrite can_draw_indexed to use can_draw

Comment (lachlansneff) [reply crates/bevy_render/src/render_graph/nodes/pass_node.rs]: Great idea!

**Comment by cart** [comment]
Looks good to me. Thanks @lachlansneff and @multun !

**Pull request merged**